Source/WebCore/workers/service/server/SWServer.cpp

```diff
@@ -1906,6 +1910,17 @@ Ref<BackgroundFetchEngine> SWServer::protectedBackgroundFetchEngine()
     return backgroundFetchEngine();
 }
 
+bool SWServer::addIdleHandler(CompletionHandler<void()>&& completionHandler)
```

Comment:
The meaning of "Idle" is a bit unclear here, I think we should rename the function to indicate what you mean by idle.